frontend/src/components/Timer/Timer.jsx

```diff
@@ -1,21 +1,38 @@
 import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
 import { toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
 import PropTypes from "prop-types";
 
-export default function Countdown(props) {
-  const { startingMinutes = 1, startingSeconds = 30 } = props;
-  const [mins, setMinutes] = useState(startingMinutes);
-  const [secs, setSeconds] = useState(startingSeconds);
+export default function Countdown({ counter }) {
+  const [mins, setMinutes] = useState(0);
+  const [secs, setSeconds] = useState(2);
```

Comment:
Not important, but could have been done with one state only

Reply:
alright!